App/Customer/frmCustomer.cs

```diff
@@ -29,16 +34,67 @@ private void btnAdd_Click(object sender, EventArgs e)
             {
                 cmodal.ShowDialog();
             }
-            this.Show();
+            this.LoadCustomerData();
         }
 
-        private void btnEdit_Click(object sender, EventArgs e)
+        private void LoadCustomerData()
         {
-            using (CustomerModal cmodal = new CustomerModal())
+            CustomerRepository customerRepository = new CustomerRepository();
+            dgvCustomer.DataSource = customerRepository.LoadCustomerData();
+            this.dgvCustomer.Columns["Mobile Number"].DefaultCellStyle.Alignment = DataGridViewContentAlignment.MiddleRight;
+            this.dgvCustomer.Columns["Phone Number"].DefaultCellStyle.Alignment = DataGridViewContentAlignment.MiddleRight;
+        }
+
+        private void btnEdit_Click_1(object sender, EventArgs e)
+        {
+            if (selectedCustomerId != 0)
             {
-                cmodal.ShowDialog();
+                if (MessageBox.Show("Do you want to edit the selected customer?", "Edit Customer", MessageBoxButtons.YesNo, MessageBoxIcon.Question) == DialogResult.Yes)
```

Comment:
This code block is missing an opening and ending curly brace. This will result in executing the rest of the code regardless of the actions (i.e. YES or NO).